src/main/java/projet/projetstage02/threadpool/ThreadPool.java

```diff
@@ -0,0 +1,67 @@
+package projet.projetstage02.threadpool;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Queue;
+
+//Fait par Guillaume Courtemanche et modifié par Massi
+public class ThreadPool {
```

Comment:
Can you explain to me what this class does

Reply:
Multithreading, this way we can send the http response while the email is being sent at the same time. since the email notification isn't necessary for the http status we don't need to wait for its result which makes it much faster for the front-end